src/30_Advanced/Tab_Panels_with_amp-selector.html

```diff
@@ -130,7 +130,7 @@
           <div role="tabpanel" class="tabContent">Tab three content... </div>
       </amp-selector>
 
-      <button on="tap:AMP.setState({selectedTab: 'a'})">Select tab 1</button>
+      <button class="ml1" on="tap:AMP.setState({selectedTab: 'a'})">Select tab 1</button>
```

Comment:
I find the asymmetry in class definitions disturbing (yes, I'm weird). How about wrapping all of them in a div which sets the margin?

```html
<div class="m1">
   <button on="tap:AMP.setState({selectedTab: 'a'})">Select tab 1</button>
   <button on="tap:AMP.setState({selectedTab: 'b'})">Select tab 2</button>
   <button on="tap:AMP.setState({selectedTab: 'c'})">Select tab 3</button>
</div>
```

Another option would be to add the margin after the on tap action ;-):

```html
<button on="tap:AMP.setState({selectedTab: 'a'})"  class="ml1">Select tab 1</button>
<button on="tap:AMP.setState({selectedTab: 'b'})">Select tab 2</button>
<button on="tap:AMP.setState({selectedTab: 'c'})">Select tab 3</button>
```